partner_financial_risk/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 {
     'name': 'Partner Financial Risk',
     'summary': 'Manage partner risk',
-    'version': '9.0.1.0.1',
+    'version': '9.0.1.0.2',
```

Comment:
I think it should be `9.0.1.1.0`, since it requires a module update.

Reply:
Not really. That attribute gets refreshed with a service restart.